src/ResultSet.php

```diff
@@ -18,18 +18,23 @@ final class ResultSet implements IteratorAggregate
      * @var Result[]
      */
     private array $results = [];
+    private bool $isValid = true;
+
+    public function addResult(string $attribute, Result $result): void
+    {
+        if (!$result->isValid()) {
+            $this->isValid = false;
```

Comment:
Please move it lower to 37 line

Reply:
This won't work if the first result is error